graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -291,12 +383,19 @@ def _ensure_no_cascading_type_suppressions(
                     (union_member.name.value for union_member in union_type.types)
                 )
             error_message_components.append(
-                "To fix this, you can suppress the union as well by adding union_type: None to the "
-                "type_renamings argument when renaming types, for each value of union_type "
-                "described here. Note that adding suppressions may lead to other types, fields, "
-                "etc. requiring suppression so you may need to iterate on this before getting a "
-                "legal schema."
+                "A schema containing a union with no members is invalid. To fix this, suppress the "
+                "unions using the type_renamings argument of rename_schema."
             )
+        if visitor.types_to_suppress:
+            error_message_components.append(
+                f"The following types have no non-suppressed fields, which is invalid: "
+                f"{sorted(visitor.types_to_suppress)}. To fix this, suppress the types using the "
+                f"type_renamings argument of rename_schema."
```

Comment:
Have in mind that there might be an UI built on top of this function. So python arguments might not be a thing for the user. I think that error messages can be improved in the future and this is fine enough for now. So just have this in mind.

Reply:
Hmm, I don't quite follow-- what arguments are we talking about, and why might they be unavailable for UIs?

Reply:
Imagine someone built a graphical UI on top of this function. The UI would ultimately use this function underneath to rename the schema and report any errors. When the user sees `some_python_argument` in the error message they might not be know what the error message refers to since the UI from the perspective is purely graphical. So `some_python_argument` is not even a thing from their perspective. Here an example of `some_python_argument` would be `type_renamings`. In this case, I think that the user of the graphical UI would still be able to get what's wrong based on the error message. I think that this holds true for most of your error messages, so I don't think that it's worth spending time on this. But I just wanted to point this out so that you have this in mind in the future.